tests/ha/hawk_gui.pm

```diff
@@ -35,6 +35,10 @@ sub run {
     my ($self) = @_;
     my $cluster_name = get_cluster_name;
 
+    # For facilitating the QEM process, the hawk client system will always be a 15-SP3 version
+    # That means we need to change the version and reload the needles
+    set_var('VERSION', get_var('CLIENT_VERSION'), reload_needles => 1) if get_var('CLIENT_VERSION');
```

Comment:
Great! Could you just a `record_info` after the change? Like this: `record_info('Version', 'VERSION=' . get_var('VERSION'));`.
Just to quickly have the information that the OS version is not the same in this specific test? It could be useful in case of debugging ;-) WDYT?

Reply:
Makes sense, done [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12307/files#diff-71de3dd983d4a00c0833546851291d89df494674fde6e1743125456e8f08f555R42).